### PR TITLE
Updated "Other"

### PR DESCRIPTION
**Pull request by MattsAttack**
## Description

This PR rewords and "fattens" all of the text in the other category and updates the section headings so that the "about other" page doesn't look as goofy.

I'm gonna work on the other sections later.

---

## Type of Change

- ✨ New feature
- 🗑️ Chore

## Checklist

- [x] Read the Contributing Guidelines.
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Fill out this template.
- [ ] Log your hours.
- [ ] Check that commits follow the Angular commit convention, more or less.
- [ ] Ideally, include relevant tests that fail without this PR but pass with it (if applicable).

## Tested on

- macOS 14
- Windows 10

### CODE REVIEW

**Comment by lishaduck** [comment]
Did you mean to close this?

**Comment by MattsAttack** [comment]
Yes I'm gonna wait to merge until i finish rewriting the other sections. The other section looks 10x better now though

**Comment by lishaduck** [comment]
For the future, you can mark a branch as a draft rather than closing it if it isn't ready.

File: src/content/other/electric-cars.mdx [hunk 1]
```diff
@@ -1,15 +1,9 @@
 ---
 title: Electric Cars
 description: Transportation is one of the largest sources of carbon emissions in the US
-sectionHeader: Cars, Trucks, and Things That Go
+sectionHeader: Electric Cars
```
Comment (lishaduck) [inline src/content/other/electric-cars.mdx:4]: Do you really hate my titles that much? 😜

File: src/content/other/electric-cars.mdx [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
-Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs.
-States like California have made plans to ban the sale of all new gas powered vehicles by 2035.
-Overall, it’s a great time to invest in a electric car.
+With transportation being one of the largest sources of carbon emissions in the US, the appeal of electric cars is very real. Electric cars are much more efficient than traditional vehicles when it comes to carbon emissions per mile as they run on electricity rather than gasoline. Right now is a great time to invest in an EV in the US, as there are tons of legislation currently getting passed to promote them. Additionally, all major electric car brands now support the Tesla supercharger network, so charging is less of a worry than ever! Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs. States like California have made plans to ban the sale of all new gas-powered vehicles by 2035. Overall, it’s a great time to invest in an electric car.
```
Comment (lishaduck) [inline src/content/other/electric-cars.mdx:7]: A few wording changes, but mostly just that it should be wrapped.
I think the only major thing was the use of a ton, which you inherited from me (I think). It feels too casual. Does this version sound better?

```suggestion
With transportation being one of the largest sources of carbon emissions in the US, the appeal of electric cars is very real.
Electric cars are much more efficient than traditional vehicles when it comes to carbon emissions per mile, as they run on electricity rather than gasoline.
Right now is a great time to invest in an EV in the US, as there is a lot of legislation currently in the process of getting passed to promote them.
Additionally, all major electric car brands now support the Tesla supercharger network, so charging is less of a worry than ever!
Consumers who purchase EVs are eligible for a $7,500 tax credit on new EVs or a $4,000 tax credit for used EVs.
States like California have made plans to ban the sale of all new gas-powered vehicles by 2035.
Overall, it’s a great time to invest in an electric car.
```

File: src/content/other/hydroelectric.mdx [hunk 3]
```diff
@@ -1,14 +1,16 @@
 ---
 title: Hydroelectric Power
 description: The power of water
-sectionHeader: Hypostatic? Nope, Just Hydroelectric
+sectionHeader: Hydroelectric Power
 ---
 
 {/* Is the Hypostatic Union joke too bad? */}
```
Comment (lishaduck) [inline src/content/other/hydroelectric.mdx:7]: Glad you changed this one. I was running out of puns.

File: src/content/other/hydroelectric.mdx [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 
-Water is powerful.
-It generates a lot of power in Tennessee and Washington State.
+While not as green as other initiatives due to its environmental effects, hydropower is still a great source of electricity.
+Hydropower has been used in the US for nearly 100 years. It was a key part of New Deal legislation and new dams are still being built to this day.
+The biggest issue with hydropower though is its impact on local ecosystems. It is reported that dams may obstruct fish migration, cause floods, and more.
+Overall, hydropower is a great resource just not the one that will power a green future.
+
 {/* Is the fact of Washington State true, or am I crazy? Both? */}
```
Comment (lishaduck) [inline src/content/other/hydroelectric.mdx:14]: This is outdated.

```suggestion
```

File: src/content/other/hydroelectric.mdx [hunk 5]
```diff
@@ -1,14 +1,16 @@
...
 ---
 
 {/* Is the Hypostatic Union joke too bad? */}
 
-Water is powerful.
-It generates a lot of power in Tennessee and Washington State.
+While not as green as other initiatives due to its environmental effects, hydropower is still a great source of electricity.
+Hydropower has been used in the US for nearly 100 years. It was a key part of New Deal legislation and new dams are still being built to this day.
+The biggest issue with hydropower though is its impact on local ecosystems. It is reported that dams may obstruct fish migration, cause floods, and more.
+Overall, hydropower is a great resource just not the one that will power a green future.
```
Comment (lishaduck) [inline src/content/other/hydroelectric.mdx:12]: The end sounds like something that could be modified to enlargen the nuclear power one.

```suggestion
While not as green as other initiatives due to its environmental effects, hydropower is still a great source of electricity.
Hydropower has been used in the US for nearly 100 years. It was a key part of FDR's New Deal legislation, and new dams are still being built to this day.
The biggest issue with hydropower is its impact on local ecosystems.
It is reported that dams obstruct fish migration, cause floods, and more.
Overall, hydropower is a great renewable resource while we're still migrating to renewables, just not one that will power an even cleaner and greener future.
```

File: src/content/other/atomics.mdx [hunk 6]
```diff
@@ -1,11 +1,15 @@
 ---
 title: Nuclear Power
 description: Fission, fusion—what’s up with nuclear power
```
Comment (lishaduck) [inline src/content/other/atomics.mdx:3]: We need a way to customize the punctation. This should have a "?" at the end, but doesn't because it _sometimes_ renders as "?.".

File: src/content/other/atomics.mdx
Comment (lishaduck) [inline src/content/other/atomics.mdx:1]: Could you rename this one to `nuclear-power.mdx`?

File: src/content/other/led-lights.mdx [hunk 7]
```diff
@@ -1,10 +1,13 @@
...
+sectionHeader: LED Lights
 ---
 
-LED lights actually use 90% less energy than traditional light bulbs and last 25x longer!
-By using longer-lasting lights, you can reduce your carbon emissions.
+In the past 15 years or so LED lights have nearly replaced traditional light bulbs. 
+This is due to a couple of big factors. 
+To start, LED lights use 90% less energy than traditional light bulbs and last 25x longer! By using less energy you reduce your carbon emissions. 
+In most cases, they're the same price as traditional light bulbs so by not buying them you're basically losing money.
+In conclusion, if you haven't invested in LED light bulbs yet they're definitely worthwhile.
```
Comment (lishaduck) [inline src/content/other/led-lights.mdx:11]:
```suggestion
In the past 15 years or so, LED lights have nearly replaced traditional light bulbs due to a few factors. 
To start, LED lights use 90% less energy than traditional light bulbs and last 25x longer!
By switching to LED lights, you'd use less energy, which directly corresponds to a reduction in your carbon emissions. 
In most cases, they're the same price as traditional light bulbs, so by not buying them, you're losing money because traditional light bulbs need to get replaced more often.
In conclusion, if you haven't invested in LED light bulbs yet, you should definitely look into them; they're worthwhile.
```

File: src/content/other/solar-lawn-lights.mdx [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Lawn Lights
-description: Saving both money and effort, solar-powered lawn lights are a great investment
-sectionHeader: Solar Powered Lights–Stylish and Green
+title: Solar Power Lawn Lights
+description: Saving both money and effort, solar-powered lawn lights are a great investment for your green lawn!
```
Comment (lishaduck) [inline src/content/other/solar-lawn-lights.mdx:3]: Love it!

**Comment by MattsAttack** [comment]
cool beans

File: src/content/other/utilities.mdx [hunk 9]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Utility Companies
+title: Utility Companies Green Programs
```
Comment (lishaduck) [inline src/content/other/utilities.mdx:2]:
```suggestion
title: Utility Companies’ Green Programs
```

File: src/content/other/utilities.mdx [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
 description: Many electric companies now offer programs to exclusively buy solar
-sectionHeader: Minimal-Hassle Green Energy Straight From Your Electric Company
+sectionHeader: Utility Companies Green Programs
```
Comment (lishaduck) [inline src/content/other/utilities.mdx:4]: Same here:

```suggestion
sectionHeader: Utility Companies’ Green Programs
```

File: src/content/other/solar-lawn-lights.mdx [hunk 11]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Lawn Lights
-description: Saving both money and effort, solar-powered lawn lights are a great investment
-sectionHeader: Solar Powered Lights–Stylish and Green
+title: Solar Power Lawn Lights
```
Comment (lishaduck) [inline src/content/other/solar-lawn-lights.mdx:2]:
```suggestion
title: Solar Powered Lawn Lights
```

File: src/content/other/wind.mdx [hunk 12]
```diff
@@ -1,10 +1,13 @@
 ---
 title: Wind Power
-description: While America is lagging in wind power, we can opine to our elected officials
```
Comment (lishaduck) [inline src/content/other/wind.mdx:3]: You got rid of opine 😢

File: src/content/other/wind.mdx [hunk 13]
```diff
@@ -1,10 +1,13 @@
...
-sectionHeader: Wind Turbines—Capturing Both Our Imaginations and Energy!
+description: Wind is good for more than just cooling you off on a hot summer day
```
Comment (lishaduck) [inline src/content/other/wind.mdx:3]: Feels a little weird to me seeing as a lot of electricity goes to AC.

File: src/content/other/wind.mdx [hunk 14]
```diff
@@ -1,10 +1,13 @@
...
+sectionHeader: Wind Power
 ---
 
 While not readily available for American consumers, wind power is still a great source of green energy.
+Wind turbines create energy by transforming the kinetic energy of the spinning blades into electrical energy that can be used to power communities.
+The only carbon emissions produced by wind turbines are the ones produced during their production. Once they're up they are 100% green.
+The only real issue with wind turbines is they can sometimes disrupt local ecosystems, especially for birds, but they generally don't cause many issues.
```
Comment (lishaduck) [inline src/content/other/wind.mdx:10]:
```suggestion
Wind turbines create energy by transforming the kinetic energy of the spinning blades into electrical energy that can be used to power communities.
The only carbon emissions produced by wind turbines are the ones produced during their production.
Once they're up, they are 100% green.
The only real issue with wind turbines is they can sometimes disrupt local ecosystems, especially for birds, but national and local wildlife conservatories are actively working to diminish the effects of wind power on local wildlife. 
```

**Review by lishaduck — COMMENTED** [review]
Looking good! I left ~~some~~ quite a few review comments, but it's shaping up well. I think my only general feedback is just to think about commas and line wrapping.

I was wondering, are you customizing any of the sectionHeaders? I was trying to make it so that the headers on the index pages would be different to spice things up.

File: src/content/other/wind.mdx (hunk 13 above)
Comment (lishaduck) [reply src/content/other/wind.mdx:3]: Also, the generated code for this change wasn't committed, so thats why CI's failing, @MattsAttack.

Comment (MattsAttack) [reply src/content/other/wind.mdx:3]: by generated do you mean the js files? if so i can go commit those real quick. It might get wonky though

**Comment by MattsAttack** [comment]
> I was wondering, are you customizing any of the sectionHeaders? I was trying to make it so that the headers on the index pages would be different to spice things up.

We could change them back but it felt kind of weird for the index page as you couldn't 100% tell what some of them were

File: src/content/other/wind.mdx (hunk 13 above)
Comment (MattsAttack) [reply src/content/other/wind.mdx:3]: We can try in do this in homeroom tomorrow. Since I committed some of the stuff on my pc and some it on my mac book the js files may not be 100% updated on both. also it wont let me look at the js files on my pc

Comment (lishaduck) [reply src/content/other/wind.mdx:3]: Not the js from the mdx, but but js for categories, etc.
The pages are gitignored.

**Comment by MattsAttack** [comment]
Idk what was up with that image bug earlier. I just checked again and all looks good. Gonna merge

**Review by lishaduck — APPROVED** [review]
Thanks!
